### PR TITLE
adds teaching section page

### PR DESCRIPTION
**Pull request by mikeseibel**

### CODE REVIEW

File: myuw/urls.py [hunk 1]
```diff
@@ -118,14 +119,24 @@
...
         ),
     url(r'^choose/new', new_site, name="myuw_pref_new_site"
         ),
     url(r'^choose/legacy', old_site, name="myuw_pref_old_site"
         ),
-    url(r'^teaching/?', teaching, name="myuw_teaching_page"
+    url(r'^teaching/?$', teaching, name="myuw_teaching_page"
+        ),
+    url(r'^teaching/'
+        r'(?P<section>\d{4},[a-zA-Z]+,[\w& ]+,\d{3}\/[A-Z][A-Z0-9]?)$',
```
Comment (vegitron) [inline myuw/urls.py]: how do independent study sections fit in with this?

File: myuw/templates/teaching_section.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{% extends "base.html" %}
```
Comment (mikeseibel) [inline myuw/templates/teaching_section.html]: okey doke

File: myuw/urls.py (hunk 1 above)
Comment (mikeseibel) [inline myuw/urls.py]: yeah, independent study sections show up just like any other, but the canvas url won't be right...  working on it

Comment (mikeseibel) [inline myuw/urls.py]: i lied.  we generage the right canvas url for independent study courses.

Comment (vegitron) [inline myuw/urls.py]: gotcha, i see the canvas courses now w/ multiple students in a per-instructor independent study section.

**Pull request merged**